fast/stages/03-gke-multitenant/prod/main.tf

```diff
@@ -16,13 +16,19 @@
 
 locals {
   labels = merge(var.labels, { environment = var.environment })
-
-  _gke_robot_sa      = "serviceAccount:${module.gke-project-0.service_accounts.robots.container-engine}"
-  _cloud_services_sa = "serviceAccount:${module.gke-project-0.service_accounts.cloud_services}"
-  host_project_bindings = [
-    { role = "roles/container.hostServiceAgentUser", member = local._gke_robot_sa },
-    { role = "roles/compute.networkUser", member = local._gke_robot_sa },
-    { role = "roles/compute.networkUser", member = local._cloud_services_sa }
+  
+  _gke_robot_sas      = [
+    "serviceAccount:${module.gke-project-0.service_accounts.robots.container-engine}"
+  ]
+  _cloud_services_sas = [
+    "serviceAccount:${module.gke-project-0.service_accounts.cloud_services}"
+  ]
+  host_project_cloud_services_bindings = [ for member in local._cloud_services_sas :
```

Comment:
why do you split this in two variables? Can't you just create variable as before?